web/src/beta/features/Editor/Visualizer/CanvasArea/index.tsx

```diff
@@ -35,6 +35,7 @@ const CanvasArea: React.FC<Props> = ({ sceneId, isBuilt, inEditor }) => {
     engineMeta,
     layerSelectionReason,
     useExperimentalSandbox,
+    isVisualizerReady: _isVisualizerReady,
```

Comment:
If you want to access to the VisualizerContext at initial load, you need to wait until this state is `true`.